Exposed Fill and TabIndex props to Icons

lynnlo opened the pull request:
## 📑 Description
Exposed the fill and tabIndex props to icons.
The documentation refers to tabindex but adding the props did nothing. When I looked at the package code, I found that it wasn't defined.

## ✅ Checks
- [x] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

lynnlo committed:
Added Fill and TabIndex props to Icons

lynnlo commented:
Nevermind, it works with setContext() originally.

shinokada commented:
I have to remove tabindex. Please read the following.

```
The issue "A11y: noninteractive element cannot have nonnegative tabIndex value" is related to accessibility. In the provided Svelte code, the tabindex attribute is being set for an SVG element. However, according to accessibility guidelines, non-interactive elements such as SVG should not have a non-negative tabIndex value.
```

lynnlo commented:
Thank you for clearing things up!

On Mon, Mar 18, 2024, 7:59 AM Shinichi Okada ***@***.***>
wrote:

> I updated icons by removing fill and tabindex.
> Since all icons are outline, you don't need the fill.
> I added withEvents so that tabindex is unnecessary.
>
> Thank you for your contribution.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/shinokada/svelte-feathers/pull/1#issuecomment-2004147464>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHBXFP2UDJR3YYJMOCTMM53YY36MHAVCNFSM6AAAAABE2ON4XOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDAMBUGE2DONBWGQ>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***>
>
